### PR TITLE
README Internationalization

### LINKED ISSUE #72: 文档翻译
需要将所有文档都英文翻译下， readme 也应该国际化

### PR DESCRIPTION
**Pull request by SteNicholas**
### Motivation:

README internationalization, based on Chinese README introduction.

### Modification:

Translate Chinese README introduction.

### Result:

Fixes #72

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,45 +4,43 @@
...
-SOFAJRaft 基于 [Apache License 2.0](./LICENSE) 协议，SOFAJRaft 依赖了一些第三方组件，它们的开源协议也为 Apache License 2.0，
-另外 SOFAJRaft 也直接引用了一些开源协议为 Apache License 2.0 的代码（可能有一些小小的改动）包括：
-- [JCTools](https://github.com/JCTools/JCTools) 中的 NonBlockingHashMap/NonBlockingHashMapLong
-- [Netty](https://github.com/netty/netty) 中的 HashedWheelTimer，另外还参考了 Netty 的 Pipeline 设计
-- [Protobuf](https://github.com/protocolbuffers/protobuf) 中对 UTF8 String 高效的编码/解码
-
-
+## Overview
+SOFAJRaft is a production-level, high-performance Java implementation based on the [RAFT](https://raft.github.io/) consistency algorithm that supports MULTI-RAFT-GROUP for high-load, low-latency scenarios.
+With SOFAJRaft you can focus on your business area. SOFAJRaft handles all RAFT-related technical challenges. SOFAJRaft is very easy to use, which you can master in a short amount of time with a few examples.
```
[the remark this answers is not in the document]
Comment (masaimu) [reply README.md]: "which you can master in a short amount of time with a few examples." -> "which you will comfortably use after several attempts."
(Users just need to use SOFAJRaft. )

File: README.md [hunk 2]
```diff
@@ -4,45 +4,43 @@
...
+
+## Features
+- Leader election
+- Log replication and recovery
+- Snapshot and log compaction
+- Cluster production membership management, adding nodes, deleting nodes, replacing nodes, etc.
```
Comment (masaimu) [inline README.md]: "Cluster production membership management" -> "Cluster membership management"

File: README.md [hunk 3]
```diff
@@ -4,45 +4,43 @@
...
+- Proactively change Leader for restart maintenance, Leader load balancing, etc.
```
Comment (masaimu) [inline README.md]: Mechanism of transfer Leader for reboot, load balance scene, etc.

File: README.md [hunk 4]
```diff
@@ -4,45 +4,43 @@
...
+- Symmetric network partition tolerance
+- Asymmetric network partition tolerance
+- Fault tolerance, minority failure doesn't affect the overall availability of system
+- Manual recovery cluster available for majority failure
+- Linearizable read, ReadIndex/LeaseRead
+- Replication pipeline optimistic
```
Comment (masaimu) [inline README.md]: Replication pipeline

File: README.md [hunk 5]
```diff
@@ -4,45 +4,43 @@
...
+- Built-in performance metrics based on [Metrics](https://metrics.dropwizard.io/4.0.0/getting-started.html) class library, with rich performance statistics
+- Passed [Jepsen](https://github.com/jepsen-io/jepsen) consistency verification test
+- JRaft includes an embedded distributed KV implementation
```
Comment (masaimu) [inline README.md]: "JRaft" -> "SOFAJRaft"

Comment (masaimu) [reply README.md]: "KV" -> "KV storage"

File: README.md [hunk 6]
```diff
@@ -4,45 +4,43 @@
...
+
+## Requirements
+Build-time requirement: JDK 8 or above and Maven 3.2.5 or above.
```
Comment (masaimu) [inline README.md]: "Compile requirement: JDK 8+ and Maven 3.2.5+ ."

File: README.md [hunk 7]
```diff
@@ -4,45 +4,43 @@
...
+## Documents
+- [User Guide](https://github.com/sofastack/sofa-jraft/wiki)
+- [Counter Example Details](https://github.com/sofastack/sofa-jraft/wiki/Counter-%E4%BE%8B%E5%AD%90%E8%AF%A6%E8%A7%A3)
+- [Release Notes](https://github.com/sofastack/sofa-jraft/wiki/%E7%89%88%E6%9C%AC%E5%8F%91%E8%A1%8C%E6%97%A5%E5%BF%97)
+
+## Contribution
+[How to contribute](https://github.com/sofastack/sofa-jraft/wiki/%E5%A6%82%E4%BD%95%E5%8F%82%E4%B8%8E-SOFAJRaft-%E4%BB%A3%E7%A0%81%E8%B4%A1%E7%8C%AE)
+
+## Acknowledgement
+SOFAJRaft was ported from Baidu's [braft](https://github.com/brpc/braft) and made some optimizations and improvements. Thanks to the Baidu braft team for opening up such a great C++ RAFT implementation.
```
Comment (masaimu) [inline README.md]: "SOFAJRaft was ported from Baidu's [braft](https://github.com/brpc/braft) and made some optimizations and improvements." -> "SOFAJRaft was ported from Baidu's [braft](https://github.com/brpc/braft) with some optimizing and improvement."

File: README_zh_CN.md [hunk 8]
```diff
@@ -0,0 +1,47 @@
...
+- 主动变更 Leader，用于重启维护，Leader 负载平衡等
+- 对称网络分区容忍性
+- 非对称网络分区容忍性
+- 容错性，少数派故障，不影响系统整体可用性
+- 多数派故障时手动恢复集群可用
+- 高效的线性一致读，ReadIndex/LeaseRead
+- 流水线复制
+- 内置了基于 [Metrics](https://metrics.dropwizard.io/4.0.0/getting-started.html) 类库的性能指标统计，有丰富的性能统计指标
+- 通过了 [Jepsen](https://github.com/jepsen-io/jepsen) 一致性验证测试
+- JRaft 中包含了一个嵌入式的分布式 KV 实现
```
Comment (masaimu) [inline README_zh_CN.md]: SOFAJRaft 中包含了一个嵌入式的分布式 KV 存储实现

File: README.md [hunk 9]
```diff
@@ -6,26 +6,26 @@
 
 ## Overview
 SOFAJRaft is a production-level, high-performance Java implementation based on the [RAFT](https://raft.github.io/) consistency algorithm that supports MULTI-RAFT-GROUP for high-load, low-latency scenarios.
-With SOFAJRaft you can focus on your business area. SOFAJRaft handles all RAFT-related technical challenges. SOFAJRaft is very easy to use, which you can master in a short amount of time with a few examples.
+With SOFAJRaft you can focus on your business area. SOFAJRaft handles all RAFT-related technical challenges. SOFAJRaft is very user-friendly, which you will comfortably use after several attempts.
```
Comment (masaimu) [inline README.md]: SOFAJRaft is very user-friendly, which provides several examples, making it easy to understand and use.

Comment (fengjiachun) [reply README.md]: 恩 感觉这个好一些

Comment (SteNicholas) [reply README.md]: @masaimu @fengjiachun I have already modified this description according to your comments.

**Pull request merged**